Assurance Cases.md

```diff
@@ -59,14 +59,18 @@ The final gap that I noticed was with evidence E6. [Fail2Ban]( https://pimylifeu
 #### [Return to top](#assurance-case-list)
 
 ### Assurance Case 4
-#### .red[TODO] Insert Assurance Case 4 Name
+#### Home Assistant efficiently prevents users from installing malicious addons.
 
-![Assurance Case 4 Diagram:](/images/Assurance_Case_4.PNG)
-.red[TODO] Insert Assuracne Case 4 Image
+![Assurance Case 4 Diagram:](/images/Assurance_Case_4.png)
 
 #### Evidence for Assurance Case 4
+Home Assistant offers both officially supported add-ons as well as third-party add-ons to fully utilize the capabilities of home automation with Home Assistant. They encourage users to develop their own add-ons if there are not any officially supported or third-party add-ons that support devices that users want. They also encourage those who have created add-ons to contribute to the community by publishing them to the public. Home Assistant has even created a tutorial document for advanced users who want to begin [creating and publishing add-ons](https://developers.home-assistant.io/docs/add-ons). This feature is a major aspect of Home Assistant's usability and integration ability with home automation, but this also creates a major risk to the security of the application with the possiblity of malicious add-ons.
```

Comment:
Fix the spelling to possibility**

Reply:
Not pressing, but maybe reword the last sentence? This part in particular reads a little weird. 

`but this also creates a major risk to the security of the application with the possibility of malicious add-ons.`